pyproject.toml

```diff
@@ -103,6 +104,7 @@ extend-select = [
     "W",            # pycodestyle warnings
 ]
 
+
```

Comment:
Not sure how that slipped in, thanks!

Reply:
(It's probably because I always put two blank spaces between differing tools. But I'm probably going to have to give that up after pyproject-fmt 2 comes out and I probably start using it)